docs/deployment/k8s-installation.md

```diff
@@ -0,0 +1,280 @@
+```sh
+# Set the current Racetrack's remote address
+racetrack set remote $RT_HOST/lifecycle
+# Login to Racetrack (use your Auth Token)
+racetrack login eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzZWVkIjoiY2UwODFiMDUtYTRhMC00MTRhLThmNmEtODRjMDIzMTkxNmE2Iiwic3ViamVjdCI6ImFkbWluIiwic3ViamVjdF90eXBlIjoidXNlciIsInNjb3BlcyI6bnVsbH0.xDUcEmR7USck5RId0nwDo_xtZZBD6pUvB2vL6i39DQI
+# Activate python3 job type in the Racetrack - we're gonna deploy Python jobs
+racetrack plugin install github.com/TheRacetrack/plugin-python-job-type
+# Activate kubernetes infrastructure target in the Racetrack
+racetrack plugin install github.com/TheRacetrack/plugin-kubernetes-infrastructure
+```
+
+## Deploy a first job
+
+Let's create a model which purpose is to add numbers.
+Let's keep it in a `adder` directory.
+
+Create `adder/entrypoint.py` file with your application logic:
+```python
+class Entrypoint:
+    def perform(self, numbers) -> float:
+        """Add numbers"""
+        return sum(numbers)
+```
+
+And a `adder/job.yaml` file describing what's inside:
+```yaml
+name: adder
+owner_email: sample@example.com
+lang: python3:latest
+
+git:
+  remote: https://github.com/TheRacetrack/samples
+  directory: adder
+
+python:
+  entrypoint_path: 'entrypoint.py'
+  entrypoint_class: 'Entrypoint'
+```
+
+Remember to put your git remote URL in `git.remote` field and push your changes to it.
+
+Finally, submit your job to Racetrack:
+```shell
+racetrack deploy adder
+```
+
+This will convert your source code to a REST microservice workload, called **Job**.
+
+Alternatively, you can deploy a sample from a root of the [Racetrack's repository](https://github.com/TheRacetrack/racetrack):
+```shell
+racetrack deploy sample/python-class
+```
+
+## Call your Job
+
+Go to the Dashboard at `$RT_HOST/dashboard` to find your job there.
+
+Also, you should get the link to your job from the `racetrack` client's output.
+Check it out at `$RT_HOST/pub/job/adder/latest`.
+This opens a SwaggerUI page, from which you can call your function
+(try `/perform` endpoint with `{"numbers": [40, 2]}` body).
+
+![](../assets/swaggerino.png)
+
+You can do it from CLI with an HTTP client as well:
+```shell
+curl -X POST "$RT_HOST/pub/job/adder/latest/api/v1/perform" \
+  -H "Content-Type: application/json" \
+  -H "X-Racetrack-Auth: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzZWVkIjoiY2UwODFiMDUtYTRhMC00MTRhLThmNmEtODRjMDIzMTkxNmE2Iiwic3ViamVjdCI6ImFkbWluIiwic3ViamVjdF90eXBlIjoidXNlciIsInNjb3BlcyI6bnVsbH0.xDUcEmR7USck5RId0nwDo_xtZZBD6pUvB2vL6i39DQI" \
+  -d '{"numbers": [40, 2]}'
+# Expect: 42
+```
+
+Congratulations, your Racetrack Job is up and running!
+
+## Troubleshooting
+
+Use one of these tools to inspect your cluster resources:
+
+- `kubectl`
+- Cloud Console
+- [Kubernetes Dashboard](#deploy-kubernetes-dashboard)
+- [k9s](https://github.com/derailed/k9s)
+
+- Check what resources you're actually trying to deploy with `kubectl kustomize kustomize/aks`
+
+### Deploy Kubernetes Dashboard
```

Comment:
Is this the purview of racetrack? That is, can we not expect that if someone is deploying on kubernetes and want a ui troubleshooting thing of some sort, they are going to do that for themselves?

This seems out of scope for the guide to me.

Reply:
You're right. It was more like the notes for myself, I'll cut it out.